router/core/src/main/scala/io/buoyant/router/Router.scala

```diff
@@ -302,6 +281,23 @@ object StackRouter {
       val default: PerClientParams = PerClientParams(Seq.empty)
     }
 
+    case class PathParams(prefix: PathMatcher, mk: Map[String, String] => Stack.Params)
```

Comment:
These are the values for the captured path segments from the prefix matcher.  E.g. if the prefix is `/svc/{service}` and the path is `/svc/foo` then this map is populated with `"service" -> "foo"`.  None of the service configs use this yet, but I included it to be consistent with client configs.